src/tests/stub_libsslsquid.cc

```diff
@@ -76,5 +76,13 @@ void setClientSNI(SSL *, const char *) STUB
 SBuf GetX509PEM(X509 *) STUB_RETVAL(SBuf())
 } //namespace Ssl
 
+#include "ssl/MemStats.h"
+namespace Ssl
+{
+void MemStats::dump(StoreEntry &) STUB
+MemStats &MallocStats() STUB_RETREF(MemStats)
+MemStats &ReallocStats() STUB_RETREF(MemStats)
```

Comment:
Please define these using `Ssl::` prefixes rather than "old style" `namespace Ssl {}` wrapper that we now tend to use for declarations.